app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -59,12 +57,16 @@ class BubbleBuilder extends ConsumerWidget {
             ? child
             : SwipeTo(
                 onLeftSwipe: (DragUpdateDetails details) {
-                  chatInputNotifier.setReplyToMessage(message);
```

Comment:
Does this approach accounts as a bug ?

Reply:
probably. the problem is that I do not for sure know what `watch` on a `notifier` even does... but generally we should reduce all `watch` to the specific minimums required and the recommendation for notifier is to `ref.read` when you need them so I changed it to that.

Reply:
Yeah, I'm also unsure about ```watch``` behaviour on ```notifier``` but to be on safe-side, I think this is good enough 👍🏻 !